src/main/java/com/zalando/ECommerceManagement/userManagement/controller/UserController.java

```diff
@@ -0,0 +1,31 @@
+package com.zalando.ECommerceManagement.userManagement.controller;
+
+import com.zalando.ECommerceManagement.userManagement.model.User;
+import com.zalando.ECommerceManagement.userManagement.service.UserService;
+import jakarta.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@Validated
+@RequestMapping("/users")
+public class UserController {
```

Comment:
Can you please format the code for this class by clicking ⌘⌥L (macOS) / Ctrl+Alt+L (Windows/Linux